client/packages/portal-client/vite.config.ts

```diff
@@ -35,6 +35,7 @@ export default defineConfig({
 			input: path.resolve(__dirname, './src/main.tsx'),
 			output: {
 				entryFileNames: `portal-client-bundle.js`,
```

Comment:
Old bundle file name. I assume this is resolved when #250 is merged.

Reply:
will do lets merge your changes first ;)